docs/streams/upgrade-guide.html

```diff
@@ -52,6 +52,15 @@ <h1>Upgrade Guide and API Changes</h1>
         <li> restart all new ({{fullDotVersion}}) application instances </li>
     </ul>
 
+    <p>
+        Since 3.0.0 release, Kafka Streams uses a newer RocksDB version which bumped its footer version persisted on files.
+        This means that old versioned RocksDB would not be able to recognize the bytes written by that newer versioned RocksDB,
```

Comment:
nit: Should we put the summary of this paragraph in the first sentence?
As you can see, this is the 1st sentence of the above paragraph: `Upgrading from any older version to 2.8.0 is possible: ....`, which is clear that we are introducing upgrading things.

And in this paragraph, the main point we want to deliver is "Downgrading from 3.0.0 to older versions, additional works needed to be done". So, we might be able to put this main point at the front, and then explain why it needs this work, to make it clear.

What do you think? (BTW, I'm OK with current version :) )

Reply:
I think starting the paragraph with "Downgrading from 3.0.0" is a good idea since it makes immediately clear what this paragraph is about.

Reply:
Ack.